src/main/scala/com.snowplowanalytics.iglu/client/repositories/EmbeddedRepositoryRef.scala

```diff
@@ -51,31 +51,19 @@ object EmbeddedRepositoryRef {
    * @return true if this is the configuration for
    *         an EmbeddedRepositoryRef, else false
    */
-  def isEmbedded(config: JValue): Boolean =
-    (config \ "connection" \ "embedded").toSome.isDefined
+  def isEmbedded(config: Json): Boolean =
+    root.connection.embedded.json.getOption(config).isDefined
```

Comment:
Are you guys okay with *optics* dependency?

Reply:
Not that I'm really against it (though I am, sorry), but full `ResolverConfig` case class with associated decoder would be much-much better, we don't want pass around a Json, but full correct config.